features/device-config/service/src/main/java/org/opennms/features/deviceconfig/service/impl/DeviceConfigServiceImpl.java

```diff
@@ -180,6 +180,11 @@ public String getConfigType() {
                         public String getSchedule() {
                             return getKeyedString(pollerParameters, DeviceConfigConstants.SCHEDULE, DeviceConfigConstants.DEFAULT_CRON_SCHEDULE);
                         }
+
+                        @Override
+                        public String getRetentionPeriod() {
```

Comment:
Unless we are using this in UI, this may not be needed.

Reply:
Removed